acme_diags/driver/default_diags/enso_diags_model_vs_obs.cfg

```diff
@@ -0,0 +1,14 @@
+[#]
```

Comment:
For run_type = model vs obs, (i.e. enso_diags_model_vs_obs) addition to "PRECT", we would add "TAUX", "TAUY", "NET_FLUX_SRF",regressing onto nino index.

For run_type = model vs model, (i.e. enso_diags_model_vs_model.cfg),  extra variables like "CLDTOT","CLDHIGH","CLDMED","CLDLOW" are useful 

Each variable will be one entry in the .cfg file because the contour levels might need to be variable depended. We need to test on those variables that are not available from our test model data suit. I will take care of generating the 'default_diags` part